sdb/commands/zfs/btree.py

```diff
@@ -0,0 +1,60 @@
+#
+# Copyright 2019 Delphix
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+# pylint: disable=missing-docstring
+
+from typing import Iterable
+
+import drgn
+import sdb
+
+
+class Btree(sdb.Walker):
+    """ walk  zfs_btree """
```

Comment:
We just integrated the new `help` command. It'd be great if we could use this PR as an opportunity to add documentation that could be used as an example for other commands. For example, we we could use a format like this:
```
"""
walk zfs_btree

DESCRIPTION

    A (potentially multiline) description about what the command does.

EXAMPLES

    > some_command | zfs_btree
    (example output)
```

I believe this will work as-is, and at least include a short description of the command (it'll use this one line summary); but if we could add more information that'd be great, and give us an example to use later for the rest of our commands.

Reply:
Sure - I'll give it try!

Reply:
Alright - I wrote a simple help message. Let me know if you have any suggestions